### ISSUE TITLE
[0.12.0.2] unexpected fault address 0xc214c62c00

### ISSUE THREAD

**Issue by dsw6** [influxdata/influxdb#6471]
Upgraded to 12.0.2.  The system was running for about 2 days ingesting data every 5 seconds and then crashed.  I've attached the log file.  Continuous queries run every 15 and 60 minutes.

System info:

Influx 12.0.1
Ubuntu 14.04.4 LTS

snippet from log file:
--------------------------------
```
unexpected fault address 0xc214c62c00
fatal error: fault
[signal 0xb code=0x2 addr=0xc214c62c00 pc=0xc214c62c00]

goroutine 32721 [running]:
runtime.gothrow(0xb25d40, 0x5)
	/usr/local/go/src/runtime/panic.go:503 +0x8e fp=0xc229fb0bf8 sp=0xc229fb0be0
runtime.sigpanic()
	/usr/local/go/src/runtime/sigpanic_unix.go:29 +0x265 fp=0xc229fb0c48 sp=0xc229fb0bf8
created by github.com/influxdata/influxdb/tsdb/engine/tsm1.(*Engine).Open
	/root/go/src/github.com/influxdata/influxdb/tsdb/engine/tsm1/engine.go:164 +0x251
```
[influxd.log.gz](https://github.com/influxdata/influxdb/files/237204/influxd.log.gz)

**Comment by e-dard** [comment]
Similar to #6442

**Comment by jwilder** [comment]
The line that triggered the panic is just starting a goroutine on when open the engine.  I think this might be a go 1.4.3 problem.

**Comment by jwilder** [comment]
This should be fixed when #6534 is merged.
